test_conformance/extensions/cl_khr_external_semaphore/main.cpp

```diff
@@ -14,6 +14,7 @@
 //
 #include "procs.h"
 #include "harness/testHarness.h"
+#include "harness/testHarness.h"
```

Comment:
We don't need to include the same file twice.

```suggestion
#include "harness/testHarness.h"
```

Reply:
Done